package.json

```diff
@@ -14,7 +14,7 @@
     "test:watch": "vitest"
   },
   "dependencies": {
-    "@material-tailwind/react": "2.1.9",
+    "@material-tailwind/react": "2.0.5",
```

Comment:
why downgrade?

Reply:
Unfortunately the package has poor maintenance and versions higher than `2.0.5` breaks TypeScript support :/